paddle/fluid/framework/ir/mkldnn/conv_bias_mkldnn_fuse_pass.cc

```diff
@@ -203,6 +271,16 @@ phi::DenseTensor tensor_apply_eltwise(const phi::DenseTensor& vec_a,
 }
 
 void ConvBiasFusePass::ApplyImpl(ir::Graph* graph) const {
+  FuseConvBias(graph, type(), fused_type());
+  if (type() != fused_type()) {
+    // Is the second pass useful?
+    FuseConvBias(graph, fused_type(), fused_type());
+  }
```

Comment:
What's the purpose of executing this fuse pass twice?

Reply:
After `conv2d` is split, we need to modify the related passes. The following case may be exist:
```
// Bebore this PR:
conv2d -> op1 -> elementwise_add   --- pass1--->  conv2d -> elementwise_add  ---conv_bias_mkldnn_fuse_pass--->  conv2d

// After conv2d is split: 
conv2d -> op1 -> elementwise_add   --- pass1--->  conv2d -> elementwise_add  ---conv_bias_mkldnn_fuse_pass---> fused_conv2d

conv2d -> op2 -> elementwise_add   --- pass2--->  fused_conv2d -> elementwise_add  ---conv_bias_mkldnn_fuse_pass---> fused_conv2d
```

Currently, the conv2d can be used recursively in the fuse pass, I'm not sure whether it's a way to optimize performance, but in order to keep performance, `conv_bias_mkldnn_fuse_pass` need to support `conv2d` and `fused_conv2d` as input operator. So the first time of fuse pass is processing `conv2d` and the second time of fuse pass is  processing `fused_conv2d`.

Reply:
Ok, that makes sense 👍 Thank you